packages/node_modules/@ciscospark/plugin-conversation/src/share-activity.js

```diff
@@ -333,8 +333,8 @@ ShareActivity.create = function create(conversation, object, spark) {
     parent: spark
   });
 
-  if (files) {
-    files.forEach((file) => share.add(file));
+  if (files && files.items) {
+    files.items.forEach((file) => share.add(file));
```

Comment:
Is there a concern that this may break backwards compatibility for others using `.create()` if they expect the `files` object to not have a separate `items` object? As I understand it, this is how the `conversation` service returns files in the `activities` list.